src/rules/ResourceTargetChecker.js

```diff
@@ -75,15 +76,16 @@ class ResourceTargetChecker extends Rule {
         function checkString(re, src, tar) {
             re.lastIndex = 0;
             let matches = [];
-            const strippedTar = stripPlurals(tar);
+            const strippedTar = stripPlurals(tar); // @TODO fix: this replaces all whitespaces with regular space so the whitespace matching rule does not work as intended
```

Comment:
@ehoogerbeets I'll need some context why stripPlurals also unifies all whitespace characters of the processed string and if we can remove this unification. Otherwise, this rule can't be implemented as declarative

Reply:
As discussed, let's take out the whitespace stripping part.

Reply:
I actually refactored `stripPlurals()` in  and fixed it so that now its unit tests pass without the extra stripping, see cf83094